app/services/WorkersServices.php

```diff
@@ -0,0 +1,59 @@
+<?php
```

Comment:
based on our discussion from yesterday:
1) WorkersServices.php -> service(singular)
2) is already in a service directory is clear that is service. Not needed to also name the class service